src/lib/utils/application-commands/ApplicationCommandRegistry.ts

```diff
@@ -405,6 +405,7 @@ export class ApplicationCommandRegistry {
 		guildId?: string
 	) {
 		try {
+			// @ts-expect-error Stupid version clashing
```

Comment:
This is not "stupid version clashing", this is a window into what I keep mentioning over and over and over again in prof+ chat that packages should not be published until issues like this are properly resolved and I keep being completely ignored by you all.

Reply:
Btw that's just me grumbling. You don't need to do anything with this.

Reply:
```
 Type 'import("/Users/vlad/Development/Sapphire/framework/node_modules/discord-api-types/rest/v9/interactions").RESTPostAPIChatInputApplicationCommandsJSONBody' is not assignable to type 'import("/Users/vlad/Development/Sapphire/framework/node_modules/discord.js/node_modules/discord-api-types/rest/v9/interactions").RESTPostAPIChatInputApplicationCommandsJSONBody'.
```

Nested -types 😢